homeassistant/components/wiz_light/bulblibrary.yaml

```diff
@@ -0,0 +1,197 @@
+# Bulb Library for importing possible bulb types
```

Comment:
Device specific info should be part of a separate 3rd party library published on PyPI.

Reply:
Was moved to the library and will be removed in the integration.